src/elm/Page/Community/Transfer.elm

```diff
@@ -371,11 +376,19 @@ update msg model ({ shared } as loggedIn) =
                     model |> UR.init
 
         EnteredMemo value ->
+            let
+                limitedMemo =
+                    if String.length value < 255 then
+                        value
+
+                    else
+                        String.slice 0 255 value
+            in
```

Comment:
Since you're using HTML's `maxlength` on the `textarea` under `viewForm`, you don't really need this - you could just use `value` directly (or you could not use `maxlength` and use this, I guess it doesn't really matter)